app/src/main/java/com/snatik/matches/common/Memory.java

```diff
@@ -3,26 +3,81 @@
 import android.content.Context;
 import android.content.SharedPreferences;
 import android.content.SharedPreferences.Editor;
+import android.preference.PreferenceManager;
+import android.util.Log;
+
+import java.util.Map;
+
+import static android.R.attr.id;
```

Comment:
please make sure to remove unused imports. You have this option in Android Studio. Please remove it.